courses/templates/course_detail.html

```diff
@@ -3,34 +3,44 @@
 
 {% block title %}MIT xPro | {{ course.title }}{% endblock %}
 
+{% block extrahead %}
+{{ block.super }}
+<style>
+  .header-block {
+    background-image: url('{{ course.background_image_mobile_url }}');
+  }
+
+  @media screen and (min-width: 1024px) { 
+    .header-block {
+      background-image: url('{{ course.background_image_url }}'); 
+    }
+  }
+</style>
+{% endblock %}
+
 {% block content %}
-<div class="course-detail">
-  <h1>{{ course.title }}</h1>
-  <h3>{{ course.description }}</h3>
-  <h3>Course Runs:</h3>
-  {% for course_run in course.courserun_set.all %}
-    <div style="background-color: {% cycle "#eee" "#fff" %}; padding: 10px;">
-      <h4 style="margin: 0 0 10px 0; padding: 0;">
-        {% if course_run.courseware_url %}
-            &#11016;
-            <a
-              href="{{ course_run.courseware_url }}"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              {{ course_run.title }}
-            </a>
-        {% else %}
-          {{ course_run.title }}
+<div class="header-block">
+  <div class="container header-container">
+    {% if course.program %}
+    <h5 class="program-position">Course {{ course.position_in_program }} of {{ course.program.num_courses }} in the {{ course.program.title }}</h5>
```

Comment:
This is minor, and it probably matches the design, but it's worth fixing: we either need to eliminate "the" in this sentence, or add "program" to the end of the sentence. Example: `Course 1 of 4 in the Systems Engineering`. That should be `Course 1 of 4 in Systems Engineering`, or `Course 1 of 4 in the Systems Engineering program`. I think I would prefer the first option.

While you're fixing this, can you also indent this line?

Reply:
Removed the "the", line indented.